config/sidekiq.yml

```diff
@@ -4,4 +4,3 @@ production:
   :concurrency: 10
 :queues:
   - default
-  - weekly_metrics
```

Comment:
I assume you tested this. Why does deleting the queue solve the issue?

Reply:
It won't haha, I had remove it manually from sidekiq cron in order to stop running. I think it is also correct to remove it from this file just in case in the future the app is restarted and cron reads this config, isn't it?